src/com/sfuentes/NegativeToPositive.java

```diff
@@ -0,0 +1,16 @@
+package com.sfuentes;
+
+import java.util.ArrayList;
+
+public class NegativeToPositive {
+  public ArrayList<Double> convertNegativeToPositive(double[] numbers) {
+
+    ArrayList<Double> positives = new ArrayList<>();
+    for (double number : numbers) {
+      if (number < 0)
+        number = number * (-1);
```

Comment:
Here you can remove the parenthesis in (-1) and just leave the -1, they are not needed.